machine/v1beta1/types_vsphereprovider.go

```diff
@@ -59,8 +60,8 @@ type VSphereMachineProviderSpec struct {
 	// to FullClone.
 	// When LinkedClone mode is enabled the DiskGiB field is ignored as it is
 	// not possible to expand disks of linked clones.
-	// Defaults to LinkedClone, but fails gracefully to FullClone if the source
-	// of the clone operation has no snapshots.
+	// Default is FullClone, LinkedClone should be explicitly set there.
+	// If no snapshots found LinkedClone will falling back to FullClone mode.
```

Comment:
Just an update to the note to make it more succinct, wdyt?
```suggestion
	// Defaults to FullClone.
	// When using LinkedClone, if no snapshots exist for the source template, falls back to FullClone.
```